### PR TITLE
update lodash version

### PR DESCRIPTION
**Pull request by mauriciosoares**
This PR is related to this issue I was having: https://github.com/istarkov/google-map-react/issues/113

Turned out this issue was specific to my building process (some weird bug), but I think that'd be good to update lodash version to 4.x

### CODE REVIEW

File: src/google_map.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ import pick from './utils/pick';
 import raf from './utils/raf';
 import log2 from './utils/math/log2';
 
-import assign from 'lodash/object/assign';
-import isNumber from 'lodash/lang/isNumber';
+import { assign } from 'lodash/object';
```
Comment (istarkov) [inline src/google_map.js]: just `import assign from 'lodash/assign'`

Comment (istarkov) [inline src/google_map.js]: otherwise it will add to build all lodash/object

Comment (istarkov) [inline src/google_map.js]: the same for others

Comment (mauriciosoares) [inline src/google_map.js]: yeah, that makes sense.

my bad

**Comment by mauriciosoares** [comment]
@istarkov code updated.

**Comment by istarkov** [comment]
Thank you!!!

**Comment by mauriciosoares** [comment]
@istarkov could you please tell me when you publish this update?

I'll update my package.json as well :D

**Comment by istarkov** [comment]
Wait for `npm install` :-) will finish.

**Comment by istarkov** [comment]
google-map-react@0.10.6

**Comment by mauriciosoares** [comment]
thanks!

**Pull request merged**